app/league/dashboard/models.py

```diff
@@ -262,6 +272,8 @@ def episode_stats(cls, episode=None, season=None, num_players=5):
                 kyus_killed[game.black.id] = \
                     kyus_killed.get(game.black.id, 0) + 1
 
+        win_ratios = {p.id: wins[p.id] / games_played[p.id] for p in players}
```

Comment:
![image](https://user-images.githubusercontent.com/3718321/29051314-81bceafe-7b96-11e7-8264-f5a5df9d8c36.png)
oh wait no there we go 😬